src/test/java/it/gov/pagopa/cgn/portal/IntegrationAbstractTest.java

```diff
@@ -46,8 +47,8 @@ protected String getAzureConnectionString() {
     protected static class Initializer
             implements ApplicationContextInitializer<ConfigurableApplicationContext> {
 
-        static PostgreSQLContainer<?> postgres = new PostgreSQLContainer("postgres:11-alpine")
-                .withDatabaseName("integration-tests-db")
+        static JdbcDatabaseContainer<?> postgres = new PostgisContainerProvider()
+                .newInstance("11-3.1").withDatabaseName("integration-tests-db")
```

Comment:
do we have to hardcode this?

Reply:
The version is hardcoded because used only for testing reason. In addition it's the same used by Azure (updated in the next commit).